core/src/main/scala/kafka/coordinator/group/GroupCoordinator.scala

```diff
@@ -567,7 +567,7 @@ class GroupCoordinator(val brokerId: Int,
 
             // if this is the leader, then we can attempt to persist state and transition to stable
             if (group.isLeader(memberId)) {
-              info(s"Assignment received from leader for group ${group.groupId} for generation ${group.generationId}. " +
+              info(s"Assignment received from leader $memberId for group ${group.groupId} for generation ${group.generationId}. " +
```

Comment:
nit: Should we wrap the variable with single/double quote to make it clear? Ex: 
`info(s"Assignment received from leader '$memberId' for group '${group.groupId}' for generation '${group.generationId'}. `

Reply:
@showuon At the moment, none of the other log in the group coordinator does this so it feel a bit weird to do it only in this case. I prefer keeping it as it is for now.

Reply:
No problem. Thank you. :)